docs/src/tutorials/Installation_and_configuration.md

```diff
@@ -0,0 +1,330 @@
+# Getting started with thin-edge.io on a Raspberry Pi 
+
+After following this tutorial you will have an overview of the installation and configuration of thin-edge.io. As an example, a Raspberry Pi is used. This tutorial explains in small steps to reach the goal of sending data to an IoT cloud platform (in this case, Cumulocity IoT) and performing some additional device management tasks.
+
+
+## Introduction
+
+thin-edge.io is an open-source project initiated by Software AG and partners to provide a cloud-agnostic edge framework. It is much more generic than the device management agent, so it can connect to multiple IoT cloud platforms, and it allows flexible logic executed on the device. It is optimized for a very small footprint and high performance.
+
+The Raspberry PI is a relatively simple and cheap device but powerful. Therefore it is ideal for testing and try-outs and some production use cases.
+
+
+##  Prerequisite
+
+To follow this guide, you only need the following:
+- A [Cumulocity]([https://www.arduino.cc/en/software](https://www.softwareag.cloud/site/product/cumulocity-iot.html#/)) Trial tenant. 
+
+- A Raspberry Pi (3 or 4) with Raspian installed, for other boards and OS'es have a look [here](https://link_to_overview_supported_devices)
+- Updated device:
+```
+$ sudo apt-get update && sudo apt-get upgrade
+```
+
+## Steps
+
+This tutorial is divided into small steps. The first three steps are needed to install and connect to an IoT cloud platform. The last three are optional but needed to get a good overview of the capabilities of thin-edge.io. 
+
+[Step 1 Install thin-edge.io](#Step-1-Install-thin-edge.io)
+
+[Step 2 Configure and Connect to IoT Cloud](#Step-2-Configure-and-Connect-to-IoT-Cloud)
+
+[Step 3 Sending Device Data](#Step-3-Sending-Device-Data)
+
+[Step 4 Monitor your device](#Step-4-Monitor-your-device)
+
+[Step 5 Add software management](#Step-5-Add-software-management)
+
+[Step 6 Manage configuration files](#Step-5-Manage-configuration-files)
+
+[Step 7 Manage Log-Files](#Step-6-Manage-Log-Files)
+
+
+
+
+## Step 1 Install thin-edge.io
+
+There are two ways to install thin-edge.io:
+- Use a script
+- Manually
+
+The easiest way is to use the installation script with this command:
+```
+curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thin-edge_io.sh | sudo sh -s
+```
+This script will install the latest version of thin-edge.io with the following components:
+- Mosquitto
+- Command line Interface (CLI) tool
+- Tedge mapper
+
+If you want to manually install thin-edge.io or install another version, or upgrade the current version, please have a look [here](link to manual installation tutorial for more information.
+
+After a successful installation, you can now use thin-edge.io via the CLI and use the tedge commands.
+
+
+### Tedge CLI
+
+In the previous step, the CLI tool is installed, which is a very powerful
```

Comment:
You can insert a snapshot of the `tedge` help.

```shell
tedge             
tedge 0.7.7
tedge is the cli tool for thin-edge.io

USAGE:
    tedge [OPTIONS] [SUBCOMMAND]

OPTIONS:
        --config-dir <CONFIG_DIR>    [default: /etc/tedge]
    -h, --help                       Print help information
        --init                       Initialize the tedge
    -V, --version                    Print version information

SUBCOMMANDS:
    cert          Create and manage device certificate
    config        Configure Thin Edge
    connect       Connect to connector provider
    disconnect    Remove bridge connection for a provider
    help          Print this message or the help of the given subcommand(s)
    mqtt          Publish a message on a topic and subscribe a topic

```

Reply:
tedge help i added